language/de/acp/profile.php

```diff
@@ -164,8 +164,8 @@
 	'STEP_2_EXPLAIN_EDIT'			=> 'Hier kannst du einige Einstellungen für die Feldart vornehmen.<br /><strong>Bitte beachte, dass Änderungen keine Auswirkungen auf die bereits existierenden Angaben deiner Benutzer haben.</strong>',
 	'STEP_2_TITLE_CREATE'			=> 'Spezifische Optionen der Feldart',
 	'STEP_2_TITLE_EDIT'				=> 'Spezifische Optionen der Feldart',
-	'STEP_3_EXPLAIN_CREATE'			=> 'Da du mehr als ein Sprachpaket installiert hast, musst du auch Angaben für die anderen Sprachen machen. Du kannst diese Angaben auch später ergänzen, das Feld arbeit derweil mit der Standardsprache.',
-	'STEP_3_EXPLAIN_EDIT'			=> 'Da du mehr als ein Sprachpaket installiert hast, kannst du hier die Angaben für die anderen Sprachen ändern oder ergänzen. Wenn keine Angaben gemacht werden, arbeitet das Feld mit der Standardsprache.',
+	'STEP_3_EXPLAIN_CREATE'			=> 'Da du mehr als ein Sprachpaket installiert hast, musst du auch Angaben für die anderen Sprachen machen. Wenn du keine Angaben machst, werden für dieses benutzerdefinierte Profilfeld die Einstellungen für die Standardsprache verwendet. Du kannst diese Angaben auch später ergänzen.',
```

Comment:
"die Einstellungen für die Standardsprache" oder "die Einstellungen der Standardsprache"?

Reply:
in meinen Augen geht beides. Da man nicht die Sprache selbst einstellt sondern es um benutzerdefinierte Profilfelder geht, deren Einstellungen man ändert wäre ich eher für "für die".

Reply:
Letzteres klingt aber nicht so umständlich.

Reply:
Dann aber in beiden gleich
```suggestion
	'STEP_3_EXPLAIN_CREATE'			=> 'Da du mehr als ein Sprachpaket installiert hast, musst du auch Angaben für die anderen Sprachen machen. Wenn du keine Angaben machst, werden für dieses benutzerdefinierte Profilfeld die Einstellungen der Standardsprache verwendet. Du kannst diese Angaben auch später ergänzen.',
```